### ISSUE TITLE
Fatal Error: Parse Error when accessing CMS data

### ISSUE THREAD

**Issue by jordanthornquest** [webhook/webhook#45]
Hello! I am using Webhook 1.0.46.

I chose to start from scratch, aiming to keep things very simple. I am running the local server using the CLI tools, and whenever I run `wh serve` and make a change to my local files that involves interacting with data in the CMS, the local server crashes and gives me the error `Fatal Error: Parse Error`. However, once I restart the server and load up the site again, the changes I made are functional and visible.

Is this expected behavior?

**Comment by jordanthornquest** [comment]
Update: this appears to only be happening when appending `.url` to the image widget object. In other words:

* `{{ cms.about.profile_image|imageSize(200) }}` totally works!
* `{{ cms.about.profile_image.url }}` does not work.

**Comment by snide** [comment]
Curious, is this happening with all images coming from the CMS or just the one? Did you happen to try reuploading it?

**Comment by jordanthornquest** [comment]
Yes, I can confirm it happens when re-uploading an image, or uploading a different image. Also, restarting doesn't work. I must have imagined that last time!
